manifests/meilisearch.yaml

```diff
@@ -60,7 +60,9 @@ metadata:
     app.kubernetes.io/component: search-engine
     app.kubernetes.io/part-of: meilisearch
 spec:
-  replicas: 1
+  # Value 0 is bad interpreted it goes to 1 by default
+  # https://github.com/helm/helm/issues/3164
+  replicas: 0
```

Comment:
I think we should keep this as 1 by default.

Reply:
Hello, yes it's a mistake from my side when I tested, I will revert